.travis.yml

```diff
@@ -16,6 +18,13 @@ before_install:
   - sudo apt-get update -qq
   - sudo apt-get install -y liboctave-dev
   - sudo apt-get install -y octave
+  - if [ "x$OCT_DEV" = "xyes" ]; then
+        sudo apt-get install -y libarpack2-dev libgl2ps-dev libgraphicsmagick++-dev libqrupdate-dev libsuitesparse-dev;
+        wget https://s3.amazonaws.com/octave-snapshot/public/octave-ubuntu-trusty-snapshot.tar.xz;
+        sudo tar --extract --directory=/usr/local --strip-components=1 --file=octave-ubuntu-trusty-snapshot.tar.xz;
+        sudo chmod -R a+rX /usr/local/;
+        sudo ldconfig;
```

Comment:
This shouldn't be necessary either, no files are installed directly in /usr/local/lib, no need to update the cache